### DISCUSSION TITLE
Grid description field update

### DISCUSSION THREAD

**Discussion by chrisla23** [Security-Onion-Solutions/securityonion#10557] [Unsupported Versions] [answered]
How can I update the field that shows up as "Description" in the grid view?  I know it asks upon install. I did not fill it out on some of the earlier nodes and I am regretting not having it as our installation grows. 

Thanks,

-Chris

**Comment by Dustifett** [comment, marked as answer]
Hello Chris,

To update the description field ssh into your manager and navigate to /opt/so/saltstack/local/pillar/minions/
Then using your favorite text editor open the .sls file that corresponds to the node you want to update. In this file you'll see a field that looks like the following:

node_description: ' '

Simply place your desired description between the single quotes. Once a salt update has occurred you should see the description populate on your grid page.

-Dustin

**Comment by chrisla23** [comment]
Worked a treat! 

Thanks!

-Chris

**Reply by Dustifett** [reply]
You're welcome Chris!

-Dustin
